GI03FLMB/GI03FLMB_R00010_ingest.csv

```diff
@@ -0,0 +1,42 @@
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13422/*.hex,GI03FLMB-RIM01-02-CTDMOG061,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG062,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-10256/*.hex,GI03FLMB-RIM01-02-CTDMOG062,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG063,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13621/*.hex,GI03FLMB-RIM01-02-CTDMOG063,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG064,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13626/*.hex,GI03FLMB-RIM01-02-CTDMOG064,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG065,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13644/*.hex,GI03FLMB-RIM01-02-CTDMOG065,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG066,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13428/*.hex,GI03FLMB-RIM01-02-CTDMOG066,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG067,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13421/*.hex,GI03FLMB-RIM01-02-CTDMOG067,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG068,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13623/*.hex,GI03FLMB-RIM01-02-CTDMOG068,recovered_inst,Not Available,No recovered data
```

Comment:
checklist & raw data repo show recovered instrument data - I think this being commented out w 'not available/no recovered data' is a copy/paste error from R00009.